packages/monix-reactive/package.json

```diff
@@ -61,6 +61,6 @@
     "reactive streams"
   ],
   "dependencies": {
-    "monix-types": "0.0.0-development"
+    "funfix": "^6.2.2"
```

Comment:
I should really publish version 7 already, it's at release candidate. Will let you know.